test/jdk/java/time/tck/java/time/zone/TCKFixedZoneRules.java

```diff
@@ -136,7 +138,9 @@ public void test_getTransition_LDT(ZoneRules test, ZoneOffset expectedOffset) {
 
     @Test(dataProvider="rules")
     public void test_isValidOffset_LDT_ZO(ZoneRules test, ZoneOffset expectedOffset) {
-        assertEquals(test.isValidOffset(LDT, expectedOffset), true);
+        if (expectedOffset == ZoneOffset.UTC)
```

Comment:
Extremely minor but why not wrap if with `{}` for improved readability here with the comment

Reply:
Same [here](https://github.com/openjdk/jdk/pull/12346/commits/ec49ca3bc03d2e97fa0429c84290923066667871?diff=unified&w=0#diff-9e5aa282dc2d02c31e1d7c5ec8196a1d3d23c06e471d5114d0bd0c78ee4fe5f6R433) although it feels much more dangerous than the test!

Reply:
Yes, please always wrap with `{}` in java.time.*